src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,10 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        for (int i = 0;i < 3;i++) {
+            Lottery ball = new Lottery();
```

Comment:
```suggestion
        Lottery lottery = new Lottery();
        for (int i = 0; i < 3; i++) {
```
лотерея це не кулька, вона створює кульки